base_rest/__manifest__.py

```diff
@@ -6,31 +6,32 @@
     "summary": """
         Develop your own high level REST APIs for Odoo thanks to this addon.
         """,
-    "version": "15.0.1.2.0",
+    "version": "16.0.1.0.0",
     "development_status": "Beta",
     "license": "LGPL-3",
     "author": "ACSONE SA/NV, " "Odoo Community Association (OCA)",
-    "maintainers": ["lmignon"],
```

Comment:
@lmignon, as you requested you have been removed from the list of maintainers from this module. Maybe you would like to be removed from other modules like **base_rest_demo**?